action.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Test Action"
+description: "test action"
+inputs:
+  rules_db_path:
+    description: "Path to the rules database"
+    required: false
+    default: "scanners/"
+  modules_path:
+    description: "Path to the modules directory"
+    required: false
+    default: "scanners/"
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r ${{ github.action_path }}/requirements.txt
```

Comment:
This will not be stable / predictable since there is no pinning / lock file on the requirements.